packages/web/src/common/store/profile/sagas.js

```diff
@@ -324,20 +324,22 @@ function* fetchProfileAsync(action) {
       }
     }
 
-    const showArtistRecommendationsPercent =
-      getRemoteVar(DoubleKeys.SHOW_ARTIST_RECOMMENDATIONS_PERCENT) || 0
-    if (Math.random() < showArtistRecommendationsPercent) {
-      yield put(
-        artistRecommendationsActions.fetchRelatedArtists({
-          userId: user.user_id
-        })
-      )
+    if (!isNativeMobile) {
+      const showArtistRecommendationsPercent =
+        getRemoteVar(DoubleKeys.SHOW_ARTIST_RECOMMENDATIONS_PERCENT) || 0
+      if (Math.random() < showArtistRecommendationsPercent) {
+        yield put(
+          artistRecommendationsActions.fetchRelatedArtists({
+            userId: user.user_id
+          })
+        )
+      }
     }
 
-    // Delay so the page can load before we fetch mutual followers
-    yield delay(2000)
-
     if (!isNativeMobile) {
+      // Delay so the page can load before we fetch mutual followers
+      yield delay(2000)
```

Comment:
ugh this is so jank 😂 - i wonder if there's a better way now?

Reply:
So this is mobile web so will continue to ignore lmao!!

Reply:
it's not desktop web too??